falcon-taz/src/main/java/pt/haslab/taz/TraceProcessor.java

```diff
@@ -38,11 +38,8 @@
  * according to their type.
  * Created by nunomachado on 05/03/18.
  */
-public enum TraceProcessor
+public class TraceProcessor
 {
-    /* TraceProcessor is a singleton class implemented using the single-element enum type approach */
-    INSTANCE;
```

Comment:
This will cause Solver to fail. Please update files that rely on INSTANCE attribute.